applications/rubintv/templates/deployment-frontend.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "rubintv.fullname" . }}-frontend
+  labels:
+    {{- include "rubintv.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "rubintv.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.frontend.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "rubintv.selectorLabels" . | nindent 8 }}
+    spec:
+      automountServiceAccountToken: false
+      {{- with .Values.workers.imagePullSecrets }}
+      imagePullSecrets:
+      {{ toYaml . | nindent 8 }}
```

Comment:
This is the frontend but it's using workers.imagePullSecrets. I suspect the right answer here is to just move the pullSecrets configuration up a level and share it between the workers and the frontend on the grounds that we probably aren't going to maintain separate pull secrets, but if you want to keep them separate, this should be frontend.pullSecrets.

Reply:
I should lift it.